src/Commands/CopyTemplatesCommand.php

```diff
@@ -53,46 +53,82 @@ public function copyTemplates(string $site_name) {
       touch($clone_dir . "/" . $directory . "/.gitkeep");
     }
     exec("git add -f db/.gitkeep logs/.gitkeep");
-    $iterator = new \DirectoryIterator($base_dir . '/templates');
-    for ($iterator->rewind(); $iterator->valid(); $iterator->next()) {
-      if (is_file($iterator->current()->getRealPath())) {
-        switch ($iterator->current()->getFilename()) {
-          case 'settings.local.php':
-            copy(
-              $iterator->current()->getRealPath(),
-              $clone_dir . '/web/sites/default/settings.local.php'
-            );
-            break;
-
-          case '.envrc':
-            $contents = file_get_contents($iterator->current()->getRealPath());
-            $contents = str_replace('**PROJECT_NAME**', $site_name, $contents);
-            $contents = str_replace('**PROJECT_PATH**', $clone_dir, $contents);
-            $contents = str_replace(
-              '**HASH_SALT**',
-              Crypt::randomBytesBase64(55),
-              $contents
-            );
-            file_put_contents($clone_dir . '/' . $iterator->current()
-                ->getFilename(), $contents);
-            break;
-
-          default:
-            copy(
-              $iterator->current()->getRealPath(),
-              $clone_dir . '/' . $iterator->current()->getFilename()
-            );
-        }
-      }
-    }
+
+    $this->copyFrameworkFiles( $this->getFramework(), $site_name, $base_dir, $clone_dir );
     $this->processGitIgnore($clone_dir);
+
     exec('direnv allow');
     if (php_uname("s") == "Darwin") {
       exec('brew bundle install');
     }
     echo (file_get_contents($base_dir . "/docs/demigod.txt"));
   }
 
+  /**
+   * Copy the template files based on the framework.
+   *
+   * @param string $framework The CMS framework identified by getFramework().
+   * @param array ...$args Array of arguments required for copying files.
+   *              $site_name The site name that was called into copyTemplates.
+   *              $base_dir This plugin's base directory.
+   *              $clone_dir The directory the site was cloned into.
+   */
+  private function copyFrameworkFiles( string $framework, ...$args ) {
+    list( $site_name, $base_dir, $clone_dir ) = $args;
```

Comment:
❤️ this.

Reply:
Also, not sure if you know this, but as of 8.0 you can deconstruct the array the same way you would in Javascript `[$one, $two, $three] = $array;`

Reply:
This is an FYI. All this is fine. No changes.

Reply:
🤯  
I think I either didn't realize you could deconstruct spreads or I had forgotten but this is amazing. ‼️